### ISSUE TITLE
Wrong type argument during doom upgrade

### ISSUE THREAD

**Issue by eh8** [hlissner/doom-emacs#3221]
**What did you expect to happen?**
`doom upgrade` to proceed as normal

**What actually happened?**
Encountered unavoidable error message:

```
x There was an unexpected error:
  Type: wrong-type-argument
  Message:
    Wrong type argument
  Data:
    listp
    "-u"

Run the command again with the -d (or --debug) switch to enable debug
mode and (hopefully) generate a backtrace from this error:

  doom -d upgrade -p

If you file a bug report, please include it!
```

**System information:**
<details><pre>
Place the output of `M-x doom/info` or `~/.emacs.d/bin/doom info` here.
</pre></details>

<details>

```
emacs   version    26.3
        features   XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS GLIB NOTIFY ACL GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS LIBSYSTEMD LCMS2
        build      May 19, 2020
        buildopts  (--prefix=/usr --sysconfdir=/etc --libexecdir=/usr/lib --localstatedir=/var --with-x-toolkit=gtk3 --with-xft --with-wide-int --with-modules 'CFLAGS=-march=x86-64 -mtune=generic -O2 -pipe -fno-plt' CPPFLAGS=-D_FORTIFY_SOURCE=2 LDFLAGS=-Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now)
        windowsys  x
        daemonp    server-running
doom    version    2.0.9
        build      HEAD -> develop a814239ec 2020-05-26 02:30:54 -0400
        dir        ~/.doom.d/
system  type       gnu/linux
        config     x86_64-pc-linux-gnu
        shell      /bin/zsh
        uname      Linux 5.6.14-arch1-1 #1 SMP PREEMPT Wed, 20 May 2020 20:43:19 +0000 x86_64
        path       (~/.emacs.d/bin ~/.node_modules/bin /usr/local/bin /usr/local/sbin /usr/bin /opt/android-sdk/platform-tools /opt/android-sdk/tools /opt/android-sdk/tools/bin /opt/cuda/bin /opt/flutter/bin /usr/lib/jvm/default/bin /usr/bin/site_perl /usr/bin/vendor_perl /usr/bin/core_perl /opt/android-sdk/platform-tools /opt/android-sdk/tools /opt/android-sdk/tools/bin /opt/cuda/bin /opt/flutter/bin /usr/lib/jvm/default/bin /usr/bin/site_perl /usr/bin/vendor_perl /usr/bin/core_perl /usr/lib/emacs/26.3/x86_64-pc-linux-gnu/)
config  envfile    envvar-file
        elc-files  0
        modules    (:completion company ivy :ui doom doom-dashboard doom-quit hl-todo modeline ophints (popup +defaults) treemacs vc-gutter vi-tilde-fringe workspaces :editor (evil +everywhere) file-templates fold (format +onsave) multiple-cursors snippets word-wrap :emacs dired electric undo vc :term vterm :checkers syntax :tools (eval +overlay) lookup magit pdf tmux :lang (dart +flutter) emacs-lisp json javascript latex markdown org python sh :app calendar twitter :config (default +bindings +smartparens))
        packages   (n/a)
        unpin      (n/a)
        elpa       (n/a)
```
</details>

**Comment by hlissner** [comment]
Ah, my bad. This should be fixed in 894966b. Let me know if that isn't the case and I'll reopen the issue. Thanks for bringing it to my attention!

**Comment by eh8** [comment]
This works for me! Thank you for the fast fix.
